wgsl/index.bs

```diff
@@ -6419,8 +6416,9 @@ However, the result may not be the same when computed in floating point.
 The reassociated result may be inaccurate due to approximation, or may trigger
 an overflow or NaN when computing intermediate results.
 
-An implementation may reassociate and/or fuse operations if the optimization is
-at least as accurate as the original formulation.
+<dfn noexport>Fusing</dfn> occur when two floating-point operation performed in one step.
```

Comment:
The example of multiply-then-add is s less general than the original statement.  
Also, the original statement constrains accuracy.  So I think this part of the change is not good.